packages/glimmer-runtime/tests/updating-test.ts

```diff
@@ -134,6 +134,31 @@ test("updating a single trusting curly", () => {
   notStrictEqual(root.firstChild.firstChild.firstChild, valueNode, "The text node was blown away");
 });
 
+test("updating a single trusting curly with siblings", function() {
+  let value = '<b>brave new </b>';
+  let context = { value };
+  let getDiv = () => root.firstChild;
+  let template = compile('<div>hello {{{value}}}world</div>');
+  render(template, context);
+
+  equalTokens(root, '<div>hello <b>brave new </b>world</div>', 'Initial render');
+
+  rerender();
+
+  equalTokens(root, '<div>hello <b>brave new </b>world</div>', 'rerender');
+
+  context.value = 'another ';
+  rerender();
+
+  equal(getDiv().firstChild.textContent, 'hello ');
+  equalTokens(getDiv().childNodes[1].textContent, 'another ');
+  equal(getDiv().lastChild.textContent, 'world');
+
+  rerender({value});
+
+  equalTokens(root, '<div>hello <b>brave new </b>world</div>', 'rerender');
+});
```

Comment:
It does catch the error in `insertAdjacentHTML`. On initial render there is no `nextSibling`, so the bounds returned are incorrect when stored. That there is a node after didn't matter for the bug I fixed in #69. I'll add a second test anyway.